submissions/evgenii-del/html-css-popup/css/style.css

```diff
@@ -0,0 +1,296 @@
+body, * {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+a {
+    text-decoration: none;
+    color: #757575;
+    font-family: 'Roboto', sans-serif;
+}
+
+li {
+    list-style-type: none;
+}
+
+ul {
+    margin-left: 0;
+    padding-left: 0;
+}
+
+.wrapper {
+    max-width: 1200px;
+    margin: 0 auto;
+}
+
+/*Nav*/
+
+.nav {
+    padding: 10px;
+}
+
+.nav__items {
+    max-width: 100%;
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+}
+
+.nav__item {
+    margin-left: 7px;
+}
+
+.nav__item .link {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
+    height: 100%;
+    padding: 10px;
+    border-radius: 50%;
+    font-size: 13px;
+}
+
+.nav__avatar .link {
+    padding: 6px;
+}
+
+.nav__button {
+    position: relative;
+    border-radius: 50%;
+}
+
+.nav__link {
+    position: relative;
+}
+
+.nav__link > .link::after {
+    content: "";
+    position: absolute;
+    width: 100%;
+    height: 2px;
+    bottom: 3px;
+    left: 0;
+    background-color: #757575;;
+    visibility: hidden;
+    transform: scaleX(0);
+    transition: all 0.3s ease-in-out 0s;
+}
+
+.nav__link .link:hover::after {
+    visibility: visible;
+    transform: scaleX(1);
+}
+
+.nav__avatar .link:hover {
```

Comment:
You can combine these styles with the styles for `.nav__btn:focus, .nav__btn:hover` and `.nav__bell .link:hover`

Reply:
Move these styles to line 132. No need to duplicate the same styles